optimism/deploy/000_perpSetToken.ts

```diff
@@ -77,7 +79,9 @@ const func: DeployFunction = trackFinishedStage(CURRENT_STAGE, async function (h
     await usdcTokenInstance.connect(uniSigner).transfer(deployer, usdc(500));
   }
 
-  const perpV2LibAddress = await findDependency(PERPV2_LIBRARY);
+  const perpV2LibraryV2Address = await findDependency(PERPV2_LIBRARY_V2);
+  const positionV2LibAddress = await findDependency(POSITION_V2);
+  const perpV2PositionsLibAddress = await findDependency(PERP_V2_POSITIONS);
   const perpV2LeverageModuleAddress = await findDependency(PERPV2_LEVERAGE_MODULE);
   const slippageIssuanceModuleAddress = await findDependency(SLIPPAGE_ISSUANCE_MODULE);
```

Comment:
Has there been any discussion about adding a `supplyCapHook` here for production versions? If this product  becomes significant part of the Perp market, doesn't every additional issuance tend to push funding flows in the wrong direction?

Reply:
If a supply cap hook is required by IC, they would write one for themselves. We can skip it here.